refactor(entire project): add refactor/css on top on development

davidholyko opened the pull request:
<!--

👋 Thanks for submitting a PR!
A good pull request goes a long way. Before submitting this pull request, please make sure you can check the following boxes:
- [x] If this is a UI change, I have tested the change in mobile, tablet, and desktop viewports
- [x] I have performed a self-review of my own code
- [x] I have selected a title that summarizes my fix
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

Next, fill out the following sections:

-->

## Description
<!-- A description of the change and which issue or behavior is addressed.                      -->

<!-- Is this a UI change? If so, include screenshots!                                           -->

Fixes #348 
Fixes #247 
Fixes #241 


<!-- Issue -->
<!-- This project only accepts pull requests related to open issues. If an issue does not exist -->
<!-- for this change, please check out our guide at CONTRIBUTING.md to create one.              -->

## Motivation and Context
<!-- Answering the question, "why is this change necessary?" or "what problem does it solve?"   -->

## Type of change
<!-- Delete any of the following that do not apply, and put an `x` where it does!               -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change (updates to README and related files)

RajChaudhry commented on src/components/Common/subcomponents/OrganizationCardSaveButton.js:
```diff
@@ -0,0 +1,17 @@
+export const OrganizationCardSaveButton = props => {
+  const { saveExist, saveItem } = props;
+  const buttonSign = saveExist ? String.fromCharCode(0x2713) : '+';
+
```
Add animation through the onClick here. Via:

`        let classes = [
            styles['cbutton--effect-radomir__after'],
            styles['cbutton--effect-radomir__cbutton--click__after'],
            styles['cbutton__after'],
        ];

        this.setState({
            animateButtonInside: styles['animate-button-click'],
            animateButtonOutside: classes,
        });
        setTimeout(() => {
            this.setState({
                animateButtonInside: '',
                animateButtonOutside: [''],
            });
        },
            500
        );`

davidholyko replied:
i was planning on replacing the animation in a future PR. right now theres a lot of logic to produce that effect and the future PR would handle it in a cleaner way

timtheguy replied:
If you want to update the animation in a future PR, just make sure to create a corresponding issue referencing this conversation so there's no confusion as to why it happened this way!

davidholyko replied:
i added a todo item in the comments on the savebutton. will open up the issue once the PR is merged in

RajChaudhry commented on src/css/component-styles/AppContainer.scss:
```diff
@@ -19,16 +19,18 @@
+  background-color: #e6e6e6;
   bottom: 0px;
+  display: flex;
+  flex-direction: column;
```
Changing feedback-container to flex messed up the positioning of the message. I attached the picture of the extra space highlighted in red.
![edit](https://user-images.githubusercontent.com/33295600/61974242-0be45c00-afb4-11e9-97e6-a0016d9c6d9d.png)

davidholyko replied:
fixed

RajChaudhry commented on src/css/component-styles/OrganizationCard.scss:
```diff
@@ -0,0 +1,55 @@
+.organization-card-body {
+  display: flex;
+  flex-direction: column;
+  padding: $spacing-md;
```
missing `line-height: 1.5;` from the original OrganizationCardLayout.js file, May be the reason why the address part is a bit offset.

RajChaudhry replied:
unless `flex` covers that, i am not 100% sure

davidholyko replied:
fixed

davidholyko commented:
small changes UI:

* I removed the badge from the save resource button and had a color change on the button to give UI feedback

* Removed button animation. It can be re-implemented in a cleaner way in a future PR.

Core functionality is all still there and other miscellaneous css tasks can be good first PR's

timtheguy commented on docs/issue_template.md:
```diff
@@ -1,41 +0,0 @@
-<!--
```
Why are we removing this?

davidholyko replied:
good catch, this is one of the things i missed when doing the merge conflicts

davidholyko commented on .vscode/settings.json:
```diff
@@ -0,0 +1,22 @@
+{
+  "workbench.colorCustomizations": {
```
its left off now
